TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -157,12 +157,7 @@ private CorrelationClause CreateCorrelationClause(CodeElementsParser.Correlation
 
             if (context.new_column_names() != null)
             {
-                List<SqlColumnName> newColumnNamesList = new List<SqlColumnName>();
-                foreach (var columnName in context.new_column_names().new_column_name())
-                {
-                    SqlColumnName newColumnName = CreateSqlColumnName(columnName.column_name());
-                    newColumnNamesList.Add(newColumnName);
-                }
+                List<SqlColumnName> newColumnNamesList = (from columnName in context.new_column_names().column_name() where columnName.Diagnostics == null select CreateSqlColumnName(columnName)).ToList();
```

Comment:
I used " columnName.Diagnostics == null" to check  if there is an error from ANTLR .
I tried to use "columnName!=null" but it fails because with emty columnName (in the test )i found  with "<MissingUserDefinedWord>" as  userDefinedWord.

Reply:
As discussed directly, I think testing on the literal is cleaner